### PR TITLE
Improve required error message

### PR DESCRIPTION
**Pull request by rkuykendall**
**Jira:** https://thatsmighty.atlassian.net/browse/PROD-664

Examples:

-  `Required - Please input a valid value` => `This field is required`
- `Required - Please input a valid Law Firm` => `'Law Firm' is required`
- `Required - Please input a valid Enter an external system identifier` => `'Enter an external system identifier' is required`

### CODE REVIEW

File: src/utilities/fillIn.ts [hunk 1]
```diff
@@ -95,7 +95,7 @@ export function fillInFieldConfig(fieldConfig: IFieldConfigPartial): IFieldConfi
   const requiredValidationRule: undefined | { [key: string]: Rule } = fieldConfig.required
     ? {
         required: {
-          message: `Required - Please input a valid ${label || 'value'}`,
+          message: ` ${`'${label}'` || 'This field'} is required`,
```
Comment (tskela) [inline src/utilities/fillIn.ts:98]: I tried running this code with `label` as `undefined`, and the message printed is `'undefined' is required`. I'm assuming adding the quotation marks around the value of `label` causes the value to no longer be falsy.

Comment (rkuykendall) [reply src/utilities/fillIn.ts]: YUP, turns out label will always be defined, so I rethought the expected result and wrote tests.

File: test/utilities/fillIn.test.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    const required = true,
+      field = 'field';
+
+    expect(getRequiredMessage({ required, field })).toBe(`'Field' is required`);
+
+    expect(getRequiredMessage({ required, field, label: undefined })).toBe(`'Field' is required`);
+
+    expect(getRequiredMessage({ required, field, label: 'Law Firm' })).toBe(`'Law Firm' is required`);
+
+    expect(getRequiredMessage({ required, field, label: 'Law Firm', showLabel: false })).toBe(`'Law Firm' is required`);
```
Comment (tskela) [inline test/utilities/fillIn.test.ts:26]: How should `showLabel` affect the resulting message? I don't see it being used.

Comment (rkuykendall) [reply test/utilities/fillIn.test.ts:26]: It's not. I guess this is just a check against future stuff that might mess with things. Or to at least acknowledge this use case.

**Pull request merged**